gocov/convert/convert.go

```diff
@@ -142,21 +251,15 @@ func (c *converter) convertProfile(packages packagesCache, p *cover.Profile) err
 }
 
 // findFile finds the location of the named file in GOROOT, GOPATH etc.
-func findFile(packages packagesCache, file string) (filename, pkgpath string, err error) {
-	dir, file := filepath.Split(file)
-	if dir != "" {
-		dir = strings.TrimSuffix(dir, "/")
-	}
-	pkg, ok := packages[dir]
-	if !ok {
-		pkg, err = build.Import(dir, ".", build.FindOnly)
-		if err != nil {
-			return "", "", fmt.Errorf("can't find %q: %w", file, err)
-		}
-		packages[dir] = pkg
-	}
-
-	return filepath.Join(pkg.Dir, file), pkg.ImportPath, nil
+// moduleName - github.com/hihoak/gocov
+// importPath - github.com/hihoak/gocov/convert
+// packageFileName - github.com/hihoak/gocov/convert/some.go
+// returns
+// absolute path to some.go on machine and github.com/hihoak/gocov/convert
+func findFile(moduleName, packageFileName string) (string, string) {
+	importPath, _ := path.Split(packageFileName)
+	absPath, _ := filepath.Abs(strings.TrimPrefix(strings.TrimPrefix(packageFileName, moduleName), "/"))
+	return absPath, strings.TrimSuffix(importPath, "/")
 }
```

Comment:
This seems to be assuming the Go code is in the current module, is that right? Does this work if you specify `-coverpkg` with packages outside the current module?